src/Database.js

```diff
@@ -167,8 +167,10 @@ class Database {
             reject(new Error(message));
           } else {
             /* eslint-disable no-param-reassign */
-            migration.up = up.replace(/^--.*?$/gm, '').trim();     // Remove comments
-            migration.down = down.replace(/^--.*?$/gm, '').trim(); // and trim whitespaces
+            migration.up = up.replace(/^-- Up.*?$/gm, '').trim();     // Remove comments
+            migration.up = migration.up.replace(/^-+$/gm, '').trim();
+            migration.down = down.replace(/^-- Down.*?$/gm, '').trim(); // and trim whitespaces
```

Comment:
I think you don't need this `replace()`: `down` doesn't contain the comment, since we split on `/^--\s+?down/mi`